README.md

```diff
@@ -155,6 +155,7 @@ Pegasystems uses a standard naming practice of hostname/product/image:tag. Pega
 `platform/search`                               | (Download required) Deploys the required search engine for Pega Platform search and reporting capabilities. This Docker image contains Elasticsearch and includes all required plugins.| `<version>` or `<version>-YYYYMMDD`. |
 `platform-services/search-n-reporting-service`  | An image that deploys the required search engine for the latest generation of search and reporting capabilities in Pega Infinity as a backing service. This Docker image contains Pega Search and Reporting Services. For more information about this service, see [Instructions to configure the Pega backingservices](charts/backingservices/README.md). | `<version>` |
 `platform/clustering-service`                   | An image that deploys the required clustering service(hazelcast) for the caching capabilities in Pega Infinity as a backing service. This Docker image contains hazelcast clustering Service. **Note:** This feature is supported only from platform version "8.6" to deploy the infinity in client server hazelcast mode.| `<version>` |
+`constellation-messaging/docker-image`  | An image that deploys the constellation messaging service. For more information about this service, see [Instructions to configure the constellationui service](charts/backingservices/charts/constellation-messaging/README.md). | `<version>` |
```

Comment:
It will be more clear for clients if we use one description of the service and use it consistently? I think "constellation UI service" is appropriate. Make sense?

An image that deploys the constellation UI service. For more information about this service, see [Instructions to configure the constellation UI service](charts/backingservices/charts/constellation-messaging/README.md).